### PR TITLE
fix: timing issue when restarting beacons

### PR DESCRIPTION
**Pull request by zbettenbuk**
The stop function on windows seems to be sync, but the status is not updated immediately, so we need to handle it differently

### CODE REVIEW

**Comment by paweldomas** [comment]
Sorry, but I don't get it how it works or what's the fixed flow exactly from the description. Also I think  commit's message is a better place for the description rather than the PR as it will get lost here.

File: spot-electron/src/bt-beacon/daemon/win32/bluetoothd.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ class BluetoothD {
 
         this.advertiser.on('StatusChanged', ({ status }) => {
             this.status = status;
+            if (this.advertisingStatusUpdatedCallback === 'function') {
+                this.advertisingStatusUpdatedCallback(status === 2);
```
Comment (paweldomas) [inline spot-electron/src/bt-beacon/daemon/win32/bluetoothd.js]: this should probably be `status !== 2`?

Comment (zbettenbuk) [reply spot-electron/src/bt-beacon/daemon/win32/bluetoothd.js]: No, true equals running state, false equals stopped (or failed of any sort). See line 84, 85 below.

Comment (paweldomas) [reply spot-electron/src/bt-beacon/daemon/win32/bluetoothd.js]: I still think this code is too hard to understand, but I give up

**Pull request merged**